src/vs/workbench/contrib/themes/browser/themes.contribution.ts

```diff
@@ -382,6 +382,35 @@ registerAction2(class extends Action2 {
 	}
 });
 
+const SelectColorThemeByNameCommandId = 'workbench.action.selectThemeByName';
+
+registerAction2(class extends Action2 {
```

Comment:
If you register it as an action it will become part of the command palette. We don't want that as it's not something a user can invoke (an input is needed)

Instead use `CommandsRegistry.registerCommand(`